packages/extension/src/ui/action/App.vue

```diff
@@ -331,8 +334,32 @@ const setNetwork = async (network: BaseNetwork) => {
     }
   }
 };
+
+const onUpdatedBalance = async () => {
+  try {
+    const thisNetworkName = currentNetwork.value.name;
```

Comment:
@andborges just a small thing, rename this variable, example `currentNetworkName`